Procfile

```diff
@@ -1,4 +1,6 @@
 web: ./start.sh
-jobs: bundle exec bin/delayed_job run
+normal_jobs: bundle exec bin/delayed_job --pool=Default:2 run
+priority_jobs: bundle exec bin/delayed_job --pool=NSIDC --pool=LPDAAC run
+worker: bundle exec rake jobs:work
```

Comment:
I'm not 100% sure it's not necessary at all - I probably added it as it wasn't working otherwise.  In that same thread is a comment "rake jobs:work is no less production worthy than bin/delayed_job. Use what works for your use case."

If you want to try it without, test it first - though this does feel like a case of "if it's not broken..."

Reply:
That line causes 2 additional docker instances to be spun up, which consume additional resources.  We are already over allocated in ram, so that isn't good.

More concerning is that instead of a clean design with two queues, we now have a solution that we don't quite understand.  

Does "worker: bundle exec rake jobs:work" create a third queue, and receive a combination of high and default priority daacs?  If so, that defeats the priority mechanism we've implemented.

Do log messages clearly indicate when a worker is working on the third "worker" queue vs the "priority jobs" and "normal_jobs" queues?  If not, triage will unnecessarily complicated.

Reply:
https://bugs.earthdata.nasa.gov/browse/EDSC-1826

Reply:
I do not believe it "creates" a queue at all, and the log messages should always report which queue they are in (regardless of how a worker is spawned).

The memory consumption is indeed concerning; if this appears to be the culprit, then by all means - let's test without it and see how it does.